apps/api/src/lib/directus-adapter.ts

```diff
@@ -371,20 +394,24 @@ export function toCollectionWithSets(
 
 export function toCollectibleBase(
   template: DirectusCollectibleTemplate,
-  getFileURL: GetFileURL
+  getFileURL: GetFileURL,
+  locale = DEFAULT_LOCALE
 ): CollectibleBase {
-  const translation = template.translations[0]
-  invariant(
-    translation !== undefined && typeof translation !== 'number',
-    'no translation found'
-  )
+  const translation =
+    getDirectusTranslation<DirectusCollectibleTemplateTranslation>(
+      template.translations as DirectusCollectibleTemplateTranslation[],
+      `collectible ${template.id} has no translations`,
+      locale
+    )
 
   const rarity = template.rarity as DirectusRarity
-  const [rarityTranslation] = rarity?.translations || []
-  invariant(
-    rarityTranslation === undefined || typeof rarityTranslation !== 'number',
```

Comment:
Not 100% sure what's going on here, I think the invariant logic is changing  from `rarityTranslation === undefined` to `translation !== undefined` via the call to `getDirectusTranslation`

Seems like `rarityTranslation === undefined ` might have been incorrect though...

Reply:
Yeah I believe it was incorrect, it didn't make much sense as is